src/arrayMethodSort.js

```diff
@@ -3,10 +3,29 @@
 /**
  * Implement method Sort
  */
-function applyCustomSort() {
+function applyCustomSort(callback) {
```

Comment:
do you use it? remove
```suggestion
function applyCustomSort() {
```